### ISSUE TITLE
SPR not resolving repository name

### ISSUE THREAD

**Issue by kwapik** [ejoffe/spr#1]
I have installed SPR with use of deb package. This is the installed version:
```spr version : 0.2.0 : 2021-05-27T05:05:34Z : c49a3038```.

When trying to check the PRs status I get:
```bash
$ git spr --status
panic: Could not resolve to a Repository with the name '/'.

goroutine 1 [running]:
github.com/ejoffe/spr/spr.check(...)
        /Users/eitan/Code/spr/spr/spr.go:788
github.com/ejoffe/spr/spr.(*stackediff).getGitHubInfo(0xc0000c0480, 0x7d5ad0, 0xc0000a8000, 0xc0000bc050, 0x747280)
        /Users/eitan/Code/spr/spr/spr.go:489 +0xa15
github.com/ejoffe/spr/spr.(*stackediff).StatusPullRequests(0xc0000c0480, 0x7d5ad0, 0xc0000a8000)
        /Users/eitan/Code/spr/spr/spr.go:248 +0x92
main.main()
        /Users/eitan/Code/spr/cmd/spr/main.go:74 +0x3ab
```
This is worrying for two reasons:
1. Hardcoded(?) paths
2. Inability to find the repo name.

For 2. I checked the config file:
```yaml
$ cat .spr.yml 
githubRepoOwner: ""
githubRepoName: ""
requireChecks: true
requireApproval: true
showPRLink: true
```

After putting repo owner and repo name manually, the panic is not occurring.

**Comment by ejoffe** [comment]
1. interesting, I guess this has something to do with the goreleaser binary build stuff.

2. Looks like the script was not able to figure out the repo name and owner in your case, and left it blank in the config file. 
I will add a warning message when this happens. 
The script tries to figure out the repo owner and name by running a regex on 'git remote -v', Can you paste the output of this command for you.

**Comment by kwapik** [comment]
@ejoffe Ad 2.
```
$ git remote -v
origin  git@github.com:owner/repo (fetch)
origin  git@github.com:owner/repo (push)
```

**Comment by ejoffe** [comment]
That matches the regex, I wonder if maybe the git command failed (silently) for some reason and that's why the repo/owner couldn't be fetched correctly. I'll add better error handling and a warning message if owner/repo isn't found.

**Comment by kwapik** [comment]
> That matches the regex,
After double checking this, I have found some issues in the regex. Created PR to fix that.

**Comment by ejoffe** [comment]
Merged pull request and released version v0.3.1.

Eitan
